### PR TITLE
Ports the NTSS Raven from TG station. Slightly remaps delta escape / faragus escape

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Ports the NTSS Raven from TG station. 4500 credits. Well defended with AI turrets, and shield generators. If you got a shuttle approved on biohazard, or cult summoning, this is the shuttle for you.

Lowers military shuttle cost / speed, to make it the cheaper version of this shuttle, since it is much less well defended.

Adds custom shield generators to the Raven, with 4 tile range instead of 2 tile.

Adjusts delta escape / viro to fit the raven, viro now has a toxin medkit and a pet mouse, to fill the extra space.

Faragus disposals moved farther out on east side so shuttle can be boarded from east side / doesnt crush it

## Why It's Good For The Game

More shuttle options are good. This option is very expencive, but still has a reasonable ammount of room, and is very well defended against biohazards, via shields or turrets.

Turrets will not fire through shields well. As such, crew will have to chose whichever is better to keep the ship safe!

## Images of changes

68747470733a2f2f616666656374656461726330372e626c6f622e636f72652e77696e646f77732e6e65742f6d6462322f696d616765732f32373238363435322f32323536383230373639342f612f5f6d6170735f6d61705f66696c65735f73687574746c65735f656d

## Testing

Ordered shuttle on each map, confirmed it could dock right.

## Changelog
:cl:
add: Added the NTSS Raven shuttle, a heavily armed and shielded shuttle, perfect for biohazards, simple mobs, and others!
tweak: Military shuttle is slower and cheaper now.
tweak: Tweaked delta virology and escape.
tweak: Tweaked faragus escape disposals
/:cl:

### CODE REVIEW

**Review by 1080pCat — CHANGES_REQUESTED** [review]
Delta:
Theres a random stray cable here (doors removed for visibility)
image

Missing a cable in viro here
image

Stray pipes here
image

Up here too
image

Faragus:
This isn't a space facing window, shouldn't be grilled
image

Raven:
Missing wall here
image

These two tesla coils aren't anchored, and they'd probably be better as grounding rods imo
image

These crates could probably do with being the o2 ones, im not personally sure if the o2 ones already have contents however.
image
image
 
Do the docking side doors *really* need tiny fans?
image

Just a general suggestion, but for 4500 credits, maybe putting a blood bag crate in here would be a good idea?
image

**Comment by Kholdstare2** [comment]
For the medical section it would be less cluttered if the defib was inside a wall unit instead, behind the surgery table. Maybe a blood crate in its place?

**Comment by Qwertytoforty** [comment]
> For the medical section it would be less cluttered if the defib was inside a wall unit instead, behind the surgery table. Maybe a blood crate in its place?

appologies, need to update the image, there is now a blood crate and medical vendor. Defib is on the table where the syringes are, though I may mount it to a wall, yes

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -158,6 +158,19 @@
...
+/mob/living/simple_animal/mouse/white/Brain
+	name = "Brain"
+	real_name = "Brain"
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "splats"
+	unique_pet = TRUE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/mouse/brown/Brain/update_desc()
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:170]:
```suggestion
/mob/living/simple_animal/mouse/white/Brain/update_desc()
```

**Review by Henri215 — COMMENTED** [review]
Took a look on the shuttle too, its looking good

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -158,6 +158,19 @@
 	// Tom fears no cable.
 	ADD_TRAIT(src, TRAIT_SHOCKIMMUNE, SPECIES_TRAIT)
 
+/mob/living/simple_animal/mouse/white/Brain
+	name = "Brain"
+	real_name = "Brain"
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:165]:
```suggestion
```
Inherited.
And do you think the response_harm should be better as "splats" or "stamps on"?

File: code/modules/supply/supply_packs/pack_shuttle.dm [hunk 3]
```diff
@@ -83,14 +83,19 @@
...
-	cost = 4250
+	cost = 3500
 	template = /datum/map_template/shuttle/emergency/military
 
+// if you can pool enough money together for this one, maybe you deserve it
+
+/datum/supply_packs/abstract/shuttle/raven
+	cost = 4500 //I do wonder about shuttle prices, especially considering how many guns you could get for this price. Perhaps a lowering in the future.
+	template = /datum/map_template/shuttle/emergency/raven
+
```
Comment (Henri215) [inline code/modules/supply/supply_packs/pack_shuttle.dm:95]:
```suggestion
```

**Review by Henri215 — COMMENTED** [review]
Everything else is lewcing good

**Review by 1080pCat — APPROVED** [review]
lookinnnn good!

**Review by AyIong — APPROVED** [review]
Look's Good To Henri

**Review by Henri215 — APPROVED** [review]
Looks good to Aylong

**Review by Contrabang — APPROVED** [review]

**Pull request merged**